src/live-preview.ts

```diff
@@ -184,9 +184,22 @@ export default class LivePreview {
                 })`;
         }
 
+        if (!this.config.stackDetails.environment) {
+            throw `To use edit tags, you must provide the preview environment. Specify the preview environment while initializing the Live Preview SDK.
+
+                ContentstackLivePreview.init({
+                    ...,
+                    stackDetails: {
+                        environment: "Your-environment"
+                    },
+                    ...
+                })`;
+        }
+
```

Comment:
Do we have a test case for this flow? Let's check the coverage of the file

Reply:
i've updated the existing test case for this flow according to the changes